CMSgov/easi-app
# Add system route

# EASI-1483

## Changes proposed in this pull request

- Attach `SystemList` view to `/system-profile` route
- Put the `/system-profile` route under the feature flag
- Update `NavigationBar` `Systems` tab link
- Remove `SystemList` from `Sandbox`

## Reviewer Notes



## Setup & How to test

- If you don't have a local connection to LaunchDarkly configured, you can (temporarily) edit src/views/FlagsWrapper/index.tsx and set `systemProfile: true`.
- Pull the branch locally and test it
- Check that all code is adequately covered by tests
- Make comments, even if it's minor!



## PR Author Review Checklist

functions, or e2e tests as necessary
the [rotor menu](https://support.apple.com/guide/voiceover/with-the-voiceover-rotor-mchlp2719/mac)
- For any new migrations/schema changes:

## Screenshots
